### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by avbel**
* Added 1-click deplay to Azure
* Removed request of catapult phone number, callback url
* All contacts/user data can be filled on deploy (instead of editing of code)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,48 @@
 
 Catapult Api demo app (creating calls and bridges)
```
Comment (dtolb) [inline README.md]: When you see catapult mentioned in a readme as you're refurbishing the demos, can you add a link : 

http://ap.bandwidth.com/?utm_medium=social&utm_source=github&utm_campaign=dtolb&utm_content=_

File: README.md [hunk 2]
```diff
@@ -1,30 +1,49 @@
 ### csharp-call-contacts-example
 
-Catapult Api demo app (creating calls and bridges)
+[Catapult](http://ap.bandwidth.com/?utm_medium=social&utm_source=github&utm_campaign=dtolb&utm_content=_) Api demo app (creating calls and bridges)
```
Comment (dtolb) [inline README.md]: As you refactor the examples, if you catch the any mentions of catapult, can you link back with this URL: http://ap.bandwidth.com/?utm_medium=social&utm_source=github&utm_campaign=dtolb&utm_content=_

If you miss it, don't worry either

File: azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
+{
+  "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion":"1.0.0.0",
+  "parameters":{
+    "siteName":{
+      "type":"string",
+      "metadata": {
+        "description": "The name of the web app that you wish to create."
```
Comment (dtolb) [inline azuredeploy.json]: Name?

**Comment by dtolb** [comment]
I like that you removed the config part for the phone numbers.  How difficult would it be to add an optional database connector (just use memory if not configured) and have a stylized web page that allows users to enter their name/phone number instead of having that ready at time of deploy?  *Make sense?*

**Comment by avbel** [comment]
Ok. I'll add that.

2016-01-21 22:08 GMT+03:00 Daniel Tolbert <notifications@github.com>:

**Comment by avbel** [comment]
Done

**Pull request merged**